graph/encoding/dot/decode.go

```diff
@@ -39,28 +39,34 @@ type PortSetter interface {
 }
 
 // Unmarshal parses the Graphviz DOT-encoded data and stores the result in dst.
+// If the number of graphs encoded in data is not one, an error is returned and
```

Comment:
Is this useful? The caller cannot distinguish it from other error situations. Or is the intention different?

Reply:
The error is different and in the case of other errors there will be no graph written. My intention in the longer run is to use a rich error that allows the user to actually find out how many graphs there were.

I don't know how useful it is. Maybe it would be better to have something like a `Decoder` type that will iterate over the input and write graphs while they exist.

Reply:
Thanks, both SGTM.

Reply:
I can back this out and just leave the doc change and deal with what we want to do here later.

Reply:
I don't think it's necessary, just leave it in and we can still revisit this later.